furthermaths/coordinate_geometry.py

```diff
@@ -0,0 +1,39 @@
+# co ordinate geometry
+from math import sqrt ,inf
```

Comment:
```suggestion
from math import sqrt ,inf
```
the point is not to use the math function but to create all the functionality yourself

Reply:
can remove sqrt, what about infinity
will have to use `math.inf`